pkg/ccl/changefeedccl/alter_changefeed_test.go

```diff
@@ -2183,3 +2183,34 @@ func getNFromSet[K cmp.Ordered](rnd *rand.Rand, s map[K]struct{}, n int) []K {
 	})
 	return ks[:n]
 }
+
+// TODO(#147428): Break this test up, assert payloads, make sure that
```

Comment:
this todo is the same issue as this pr fixes. so lets either remove the comment, change the referenced issue, or do the thing.